code/mob/living/critter/mob_bots.dm

```diff
@@ -399,3 +399,682 @@ ABSTRACT_TYPE(/datum/targetable/critter/bot/fill_with_chem)
 				continue
 			fireflash(F,0.5,temp, chemfire = CHEM_FIRE_RED)
 
+ADMIN_INTERACT_PROCS(/mob/living/critter/robotic/securitron, proc/change_hand_item)
+/mob/living/critter/robotic/securitron
+	name = "securitron"
+	real_name = "securitron"
+#ifdef HALLOWEEN
+	desc = "A little security robot, apparently carved out of a pumpkin.  It looks...spooky?"
+	icon = 'icons/misc/halloween.dmi'
+#else
+	desc = "A little security robot.  It looks less than thrilled."
+	icon = 'icons/obj/bots/aibots.dmi'
+#endif
+	icon_state = "secbot0"
+	blood_id = "oil"
+	speechverb_say = "beeps"
+	speechverb_gasp = "warbles"
+	speechverb_stammer = "bleeps"
+	speechverb_exclaim = "boops"
+	speechverb_ask = "bloops"
+	robot_talk_understand = TRUE
+	density = FALSE
+	hand_count = 1
+	base_move_delay = 3.25
+	base_walk_delay = 4.25
+	can_burn = FALSE
+	can_grab = TRUE
+	can_disarm = TRUE
+	can_help = TRUE
+	dna_to_absorb = 0
+	metabolizes = FALSE
+	custom_gib_handler = /proc/robogibs
+	stepsound = null
+	health_brute = 25
+	health_brute_vuln = 1
+	health_burn = 25
+	health_burn_vuln = 1.25
+	ai_retaliates = FALSE // this is a lie- special retaliation behavior in was_harmed and whistles
+	ai_retaliate_persistence = RETALIATE_UNTIL_INCAP
+	ai_type = /datum/aiHolder/patroller/packet_based/securitron
+	is_npc = TRUE
+	ai_attacks_neutral = TRUE
+	var/random_name = TRUE
+	var/control_freq = FREQ_BOT_CONTROL
+	var/chase_speed_bonus = 0.9
+	var/obj/machinery/camera/camera = null
+	var/no_camera = FALSE
+	var/initial_limb = /obj/item/baton/mobsecbot
+	var/drop_limb_item = FALSE
+	var/net_id
+	var/power = TRUE
+	var/emagged = 0
+	var/emote_cooldown = 7 SECONDS
+	var/arrest_cooldown = 10 SECONDS
+	var/siren_active = FALSE
+	var/list/req_access = list(access_security)
+	var/weapon_access = access_carrypermit
+	var/contraband_access = access_contrabandpermit
+	var/check_contraband = TRUE
+	var/check_records = TRUE
+	var/is_detaining = FALSE
+	var/report_arrests = TRUE
+	var/patrolling = FALSE
+	var/lockdown = FALSE
+	var/list/datum/contextAction/contexts = list()
+	var/datum/contextLayout/configContextLayout = new /datum/contextLayout/experimentalcircle
+
+/mob/living/critter/robotic/securitron/New()
+	. = ..()
+
+	if(src.name == "securitron")
+		src.real_name = "Securitron-" + "[rand(1,9)]" + "[rand(0,9)]" + "[rand(0,9)]"
+		src.name = src.real_name
+		src.UpdateName()
+	if(src.initial_limb)
+		var/datum/handHolder/HH = hands[1]
+		src.hud.add_object(src.equipped(), HUD_LAYER+2, HH.screenObj.screen_loc)
+
+	src.net_id = generate_net_id(src)
+
+	remove_lifeprocess(/datum/lifeprocess/blindness)
+	remove_lifeprocess(/datum/lifeprocess/blood)
+	remove_lifeprocess(/datum/lifeprocess/radiation)
```

Comment:
why?

Reply:
Beepsky dying to radiation is very unintended behavior imo, and reaching 100% rad resist seems to be impossible now